classiclasagne.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div>
+        <h1><a href="./index.html">Home</a></h2>
+    </div>
+    <h2>Easy classic lasagne</h2>
+    <h2>Ingredients</h2>
+    <ul>
+      <li>1 tbsp olive oil</li>
+      <li>2 rashers smoked streaky bacon</li>
+      <li>1 onion , finely chopped</li>
+      <li>1 celery stick, finely chopped</li>
+      <li>1 medium carrot , grated</li>
+    </ul>
+    <h2>Preparation time</h2>
+    <ul>
+      <li>60 mins</li>
+    </ul>
+    <h2>Instructions</h2>
+    <ul>
+      <p>
+        Heat the oil in a large saucepan. Use kitchen scissors to snip the bacon
+        into small pieces, or use a sharp knife to chop it on a chopping board.
+        Add the bacon to the pan and cook for just a few mins until starting to
+        turn golden. Add the onion, celery and carrot, and cook over a medium
+        heat for 5 mins, stirring occasionally, until softened.
+      </p>
+      <img
+        src="https://images.immediate.co.uk/production/volatile/sites/30/2020/08/classic-lasange-4a66137.jpg?resize=960,872?quality=90&webp=true&resize=300,272"
+        alt=""
```

Comment:
I think no need to include an empty alt attribute here :)! Ideally we can describe the image here for screen readers